source/community/release_notes/palm.rst

```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
+* There is a complete list of changes in the `open edX wiki <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2022/11/21/3584589831/2U+New+Learner+Home+Page#Comparison-with-old-experience>`_. 
+
+To turn on the New Learner home page, add and enable the :code:`learner_home_mfe.enabled` Waffle Flag in the django 
+admin. To support a phased role out, there is also a setting :code:`LEARNER_HOME_MFE_REDIRECT_PERCENTAGE`. Set it to an 
+integer value (0 - 100) representing what percentage of users should be redirected to Learner Home. 0 represents no 
+rollout, 100 represents rollout to all users.
+
 
 Instructor Experiences
 **********************
 
+New Visual Problem Editor
+=========================
+
+The release includes an improved problem-authoring experience with an interactive WYSIWYG editor. Writing markdown 
+code is no longer necessary. But, the advanced mode is still available, maintaining the ability to write and edit 
+OLX XML.
+
+The open edX wiki page `[2U] New Visual Problem Editor <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2023/04/07/3724312593/2U+New+Visual+Problem+Editor>`_ 
+provides a brief explanation of what has changed in the problem editor. Updated detailed instructions on writing 
+problems can be found in section `8.4. Working with Problem Components <https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_components/create_problem.html#working-with-problem-components>`_ of the Building and Running an edX Course documentation.
+
+The visual problem editor is an update to the Course Authoring Micro-frontend first released in Olive. Assuming that 
+you are already using the Course Authoring MFE, all you need to do to enable the Visual Problem Editor is add the 
+waffle flag :code:`new_core_editors.use_new_problem_editor` and set the value to “Yes” for all users.
+
+New ORA Grading Experience
+==========================
+
+In this new on-platform grading experience one can easily preview common file types, assign rubric values, provide 
+comments, and coordinate grading with all members of the course teams. Complete documentation is in section 
+`10.26.4. Staff Grading for Open Response Assignments 
+<https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/open_response_assessments/ORA_Staff_Grading.html#staff-grading-for-open-response-assignments>`_  of the Building and Running an edX Course documentation. 
+
+The new ORA grading experience depends on the ORA Grading Micro-frontend, which was included as an experimental 
+feature in Olive. To turn on the feature, add the feature flag :code:`openresponseassessment.enhanced_staff_grader`.
+
+New Bulk Email Experience
+=========================
+
+Enabling the Communications Micro-frontend allows instructors to schedule bulk emails, instead of sending them 
```

Comment:
Is this an experimental MFE in Palm? or is it on by default?

Reply:
It is expected to be "on" by default, in the sense that it is usable without jumping through any hoops other than enabling the bulk email feature in the first place.  (This is a pre-existing flag that has been off since its implementation, years ago.)

Reply:
Ok. I didn't realize that the MFE settings are handled by the Tutor plugin: 

- https://github.com/overhangio/tutor-mfe/pull/126/files#diff-e7cc1a222153171e9e8c772d71621d1394f40264ecb772a82212c23953a01fefR12
- https://github.com/overhangio/tutor-mfe/pull/126/files#diff-9dd8a655b8597bfd6cdb64b96b2a805174561cae462059559fdc4a2b8be0ac06R51

> other than enabling the bulk email feature in the first place

@arbrandes what setting are you refering to?

Reply:
Adolfo pointed me to the settings that were documented back in the Eucalyptus release, https://docs.openedx.org/en/latest/community/release_notes/eucalyptus.html#updates-to-bulk-email -- I don't think we need to mention them here.